tools/scripts/android_helpers.py

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/python3
+
+# -----------------------------------------------------------
+# Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+# This product includes software developed at Datadog (https://www.datadoghq.com/).
+# Copyright 2023-Present Datadog, Inc.
+# -----------------------------------------------------------
+
+import datetime
+import io
+import json
+import os
+import re
+import subprocess
+import time
+from typing import Optional
+
+AVD_TAG = 'google_apis'
+AVD_ABI = 'arm64-v8a'
+
+def _get_android_home() -> str:
+    android_home = os.environ['ANDROID_HOME']
+    if android_home is None:
+        raise Exception('ANDROID_HOME is not set.')
+
+    return android_home
+
+def _get_avd_manager() -> str:
+    return os.path.join(_get_android_home(), 'cmdline-tools', 'latest', 'bin', 'avdmanager')
+
+def _get_sdk_manager() -> str:
+    return os.path.join(_get_android_home(), 'cmdline-tools', 'latest', 'bin', 'sdkmanager')
+
+def _get_emulator_command() -> str:
+    return os.path.join(_get_android_home(), 'emulator', 'emulator')
+
+def _run(args: list[str], write_std_out: bool = False) -> str:
+    process = subprocess.Popen(args,
+                               stdout=subprocess.PIPE,
+                               stderr=subprocess.STDOUT,
+                               start_new_session=True,
+                               universal_newlines=True)
+    output = io.StringIO()
```

Comment:
where is this going to appear ? what's the difference between the `print(line)` and this `io.StringIO()`